package.json

```diff
@@ -56,6 +56,7 @@
   },
   "dependencies": {
     "@slack/bolt": "^1.4.0",
+    "@slack/types": "^1.2.1",
```

Comment:
@johnkahn Not sure what was up, but my IDE was complaining about this being in `devDependencies`. Any ideas?

Reply:
Probably because they don't follow the standards, smh. Types are supposed to be built into the project or they should exist inside the DefinitelyTyped project imported with `@types/slack` (which doesn't exist)